src/Orchard.Web/Modules/Orchard.Azure.Authentication/OwinMiddlewares.cs

```diff
@@ -13,71 +16,110 @@
-
-        public OwinMiddlewares(ISiteService siteService) {
-            Logger = NullLogger.Instance;
-
-            var site = siteService.GetSiteSettings();
-            var azureSettings = site.As<AzureSettingsPart>();
-
-            _azureClientId = ((azureSettings.ClientId == null) || (azureSettings.ClientId == string.Empty)) ? 
-                "[example: 82692da5-a86f-44c9-9d53-2f88d52b478b]" : azureSettings.ClientId;
 
-            _azureTenant = ((azureSettings.Tenant == null) || (azureSettings.Tenant == string.Empty)) ? 
-                "faketenant.com" : azureSettings.Tenant;
+        private readonly string _azureClientId = DefaultAzureSettings.ClientId;
+        private readonly string _azureTenant = DefaultAzureSettings.Tenant;
+        private readonly string _logoutRedirectUri = DefaultAzureSettings.LogoutRedirectUri;
+        private readonly string _azureAdInstance = DefaultAzureSettings.ADInstance;
+        private readonly bool _azureWebSiteProtectionEnabled = DefaultAzureSettings.AzureWebSiteProtectionEnabled;
+        private readonly string _relativePostLoginCallBackUrl = "Orchard.Azure.Authentication/Account/LogOnCallBack";
 
-            _azureADInstance = ((azureSettings.ADInstance == null) || (azureSettings.ADInstance == string.Empty)) ? 
-                "https://login.microsoft.com/{0}" : azureSettings.ADInstance;
 
-            _logoutRedirectUri = ((azureSettings.LogoutRedirectUri == null) || (azureSettings.LogoutRedirectUri == string.Empty)) ? 
-                site.BaseUrl : azureSettings.LogoutRedirectUri;
+        public OwinMiddlewares(ISiteService siteService)
+        {
+            Logger = NullLogger.Instance;
 
-            _azureAppName = ((azureSettings.AppName == null) || (azureSettings.AppName == string.Empty)) ? 
-                "[example: MyAppName]" : azureSettings.AppName;
+            try
+            {
+                var settings = siteService.GetSiteSettings().As<AzureSettingsPart>();
 
-            _sslEnabled = azureSettings.SSLEnabled;
+                if (settings == null)
+                {
+                    return;
+                }
 
-            _azureWebSiteProtectionEnabled = azureSettings.AzureWebSiteProtectionEnabled;
+                _azureClientId = settings.ClientId ?? _azureClientId;
+                _azureTenant = settings.Tenant ?? _azureTenant;
+                _azureAdInstance = settings.ADInstance ?? _azureAdInstance;
+                _logoutRedirectUri = settings.LogoutRedirectUri ?? _logoutRedirectUri;
+                _azureWebSiteProtectionEnabled = settings.AzureWebSiteProtectionEnabled;
+            }
+            catch (Exception ex)
+            {
+                Logger.Log(LogLevel.Debug, ex, "An error occured while accessing azure settings: {0}");
+            }
         }
 
-        public IEnumerable<OwinMiddlewareRegistration> GetOwinMiddlewares() {
+        public IEnumerable<OwinMiddlewareRegistration> GetOwinMiddlewares()
+        {
```

Comment:
I thought I had taken care of reformatting everything. But apparently not, will get that fixed.